### DISCUSSION TITLE
suddeny difficulty deserializing

### DISCUSSION THREAD

**Discussion by JustTalkToSteve** [XiaoFaye/WooCommerce.NET#739] [Q&A]
A blazor application that launched a few years ago is suddenly having difficulty connecting.

On the local IIS Express server launched by visual studio, the application is able to connect and functions correctly.

However, the same project running on the azure server is now giving me the following error.
"One or more errors occurred. (There was an error deserializing the object of type System.Collections.Generic.List`1[[WooCommerceNET.WooCommerce.v3.Customer, WooCommerce.NET, Version=0.8.3.0, Culture=neutral, PublicKeyToken=null]]. Encountered unexpected character '<'.)"

**Comment by XiaoFaye** [comment]
Better to check what is the json output looks like.

**Reply by JustTalkToSteve** [reply]
It's working again, I didn't get a chance to grab the json
